app/services/external/gpt_service.py

```
@@ -0,0 +1,40 @@
+        )
+
+        try:
+            response = client.chat.completions.create(
+                model="gpt-4o",
+                messages=[{"role": "user", "content": prompt}],
+                temperature=0.2
+            )
+            return response.choices[0].message.content.strip()
+        except Exception as e:
+            logger.error(f"[GPTService] GPT API 호출 실패: {e}")
+            raise APIException(ErrorStatus.GPT_API_ERROR)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)